migration/v0.37/prepare/60-move-issuer-ingress-class.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+HERE="$(dirname "$(readlink -f "${0}")")"
+ROOT="$(readlink -f "${HERE}/../../../")"
+
+# shellcheck source=scripts/migration/lib.sh
+source "${ROOT}/scripts/migration/lib.sh"
+
+log_info "moving issuer ingress class keys"
+yq_move common .issuers.letsencrypt.prod.solvers[].http01.ingress.class .issuers.letsencrypt.prod.solvers[].http01.ingress.ingressClassName
+yq_move common .issuers.letsencrypt.staging.solvers[].http01.ingress.class .issuers.letsencrypt.staging.solvers[].http01.ingress.ingressClassName
+yq_move common .issuers.extraIssuers[].spec.acme.solvers[].http01.ingress.class .issuers.extraIssuers[].spec.acme.solvers[].http01.ingress.ingressClassName
+
+if [[ "${CK8S_CLUSTER}" =~ ^(sc|both)$ ]]; then
+yq_move sc .issuers.letsencrypt.prod.solvers[].http01.ingress.class .issuers.letsencrypt.prod.solvers[].http01.ingress.ingressClassName
+yq_move sc .issuers.letsencrypt.staging.solvers[].http01.ingress.class .issuers.letsencrypt.staging.solvers[].http01.ingress.ingressClassName
+yq_move sc .issuers.extraIssuers[].spec.acme.solvers[].http01.ingress.class .issuers.extraIssuers[].spec.acme.solvers[].http01.ingress.ingressClassName
+fi
+
+if [[ "${CK8S_CLUSTER}" =~ ^(wc|both)$ ]]; then
+yq_move wc .issuers.letsencrypt.prod.solvers[].http01.ingress.class .issuers.letsencrypt.prod.solvers[].http01.ingress.ingressClassName
+yq_move wc .issuers.letsencrypt.staging.solvers[].http01.ingress.class .issuers.letsencrypt.staging.solvers[].http01.ingress.ingressClassName
+yq_move wc .issuers.extraIssuers[].spec.acme.solvers[].http01.ingress.class .issuers.extraIssuers[].spec.acme.solvers[].http01.ingress.ingressClassName
+fi
```

Comment:
Indent inside if statement.

Reply:
Working on fixing some other things in the migration script too :+1: